pom.xml

```diff
@@ -126,6 +126,11 @@
             <artifactId>commons-text</artifactId>
             <version>1.6</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
Can we parameterize this to allow code bases to be using either junit 4 or junit 5?  WCM Core is using junit 5, so if we can only do one I think that's what we should default to.  But if we can make junit version configurable, that might be even better.

Reply:
Makes sense. I have made JUnit version configurable in data-config.